include/boost/flags.hpp

```diff
@@ -42,15 +42,15 @@
         olives      = boost::flags::nth_bit(3), // == 0x08
     };
     // enable Boost.Flags for pizza_toppings
-    template<> struct boost_flags_enable<pizza_toppings> : std::true_type {};
+    static constexpr bool boost_flags_enable(pizza_toppings{}) { return true; }

```

Comment:
I think the curly braces at the argument are wrong. "pizza_toppings" is the type here. Just remove the curly braces.
... but this is documentation, which is unfinished, as you said.

Reply:
you're right - I removed the curly braces